docs/USAGE-GUIDE.md

```diff
@@ -654,6 +654,18 @@ If the read end of the pipe is closed unexpectedly, writing to the pipe will rai
 a SIGPIPE signal. **s2n-tls does NOT handle SIGPIPE.** A SIGPIPE signal will cause
 the process to terminate unless it is handled or ignored by the application.
 
+**WARNING (TLS v1.2 ONLY):** Enabling Nagle's algorithm can increase handshake latency
```

Comment:
It seems like this behavior is on by default, they should be looking to disable it.
```suggestion
**WARNING (TLS v1.2 ONLY):** Not disabling Nagle's algorithm can increase handshake latency
```

Reply:
Nagle's isn't necessarily on by default and would depend on the OS and the user's setting. I chose to word it in the positive-affirmation since it's easier to read than "not disabling".

Reply:
Yeah I don't really like "not disabling". It's like, who needs this warning more, the people who enabled Nagle's and know what it is or the people who didn't enable Nagle's and don't know what it is, but it's on by default for their OS. Your code example shows how to disable it, so I would assume that's who your audience is. If you remove the code example than I think this makes more sense.